### PR TITLE
Port remote execution tests

### PR DESCRIPTION
**Pull request by oshtaier**
```
py.test /home/ashtayer/Desktop/robottelo/tests/foreman/ui_airgun/test_jobinvocation.py 
==================================================================================== test session starts =====================================================================================
platform linux -- Python 3.6.5, pytest-3.4.0, py-1.5.3, pluggy-0.6.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/ashtayer/Desktop/robottelo, inifile:
plugins: wait-for-1.0.9, services-1.2.1, mock-1.6.3
collected 2 items                                                                                                                                                                            
2018-08-31 17:17:27 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_jobinvocation.py ..                                                                                                                                       [100%]

================================================================================= 2 passed in 429.24 seconds ==================================================================
```

### CODE REVIEW

File: tests/foreman/ui_airgun/test_jobinvocation.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+        })
+        # connect to host by ip
+        Host.set_parameter({
+            'host': client.hostname,
+            'name': 'remote_execution_connect_by_ip',
+            'value': 'True',
+        })
+        hostname = client.hostname
+        with session:
+            session.organization.select(module_org.name)
```
Comment (abalakh) [inline tests/foreman/ui_airgun/test_jobinvocation.py]: nitpick: redundant line

File: tests/foreman/ui_airgun/test_jobinvocation.py [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+        location=[entities.Location(id=DEFAULT_LOC_ID)],
+        mask=settings.vlan_networking.netmask,
+        network=settings.vlan_networking.subnet,
+        network_type='IPv4',
+        remote_execution_proxy=[entities.SmartProxy(id=1)],
+        organization=[module_org],
+    ).create()
+
+
+@tier2
```
Comment (rplevka) [inline tests/foreman/ui_airgun/test_jobinvocation.py]: can we make this t3 as well, please?

File: tests/foreman/ui_airgun/test_jobinvocation.py [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+            'host': client.hostname,
+            'name': 'remote_execution_connect_by_ip',
+            'value': 'True',
+        })
+        hostname = client.hostname
+        job_template_name = gen_string('alpha')
+        with session:
+            session.organization.select(module_org.name)
+            assert session.host.search(hostname)[0]['Name'] == hostname
+            session.jobtemplate.create({
```
Comment (rplevka) [inline tests/foreman/ui_airgun/test_jobinvocation.py]: wouldn't it be better to create this using nailgun?

File: tests/foreman/ui_airgun/test_jobinvocation.py (hunk 2 above)
Comment (oshtaier) [reply tests/foreman/ui_airgun/test_jobinvocation.py]: sure, I will change

File: tests/foreman/ui_airgun/test_jobinvocation.py (hunk 1 above)
Comment (oshtaier) [reply tests/foreman/ui_airgun/test_jobinvocation.py]: I will be happy to remove that, but only if I find machine to re-test cases as so far only one machine in whole infrastructure sufficed my needs

File: tests/foreman/ui_airgun/test_jobinvocation.py (hunk 3 above)
Comment (oshtaier) [reply tests/foreman/ui_airgun/test_jobinvocation.py]: You are right, but I want to have at least one end-to-end scenario which represent real user interactions. All next test cases in that area should use nailgun

**Comment by oshtaier** [comment]
@abalakh @rplevka can we merge that PR?
there are things that can be improved, but we will do them on next phase of airgun porting tests effort. At that moment we more than happy to have at least one test ported by each component

File: tests/foreman/ui_airgun/test_jobinvocation.py (hunk 1 above)
Comment (oshtaier) [reply tests/foreman/ui_airgun/test_jobinvocation.py]: yeah, it seems to hard to find exact moment when qe6.4 machine is not occupied by any activity. Let's remove all these context selections in separate PR

**Comment by oshtaier** [comment]
Created and issue to track progress - https://github.com/SatelliteQE/robottelo/issues/6323

**Pull request merged**